pyne/tally.pyx

```diff
@@ -21,6 +24,44 @@ from pyne.utils import QAWarning
 
 warn(__name__ + " is not yet QA compliant.", QAWarning)
 
+cdef vector[int] to_vector_int(value):
+    cdef vector[int] value_proxy
+    cdef int ivalue
+    cdef int value_size
+    cdef int * value_data
+    # value is a ('vector', 'int32', 0)
+    value_size = len(value)
+    if isinstance(value, np.ndarray) and (< np.ndarray > value).descr.type_num == np.NPY_INT64:
```

Comment:
Could this conditional only choose how to set `value_data`, and then the allocation and assignment can be outside the conditional and appear once?
```
if .....
    value_data = <int * > np.PyArray.....
else:
    value_data = value
value_proxy = vector[int]....
```

Reply:
nice catch !